--- gdk-pixbuf/io-tiff.c.str	2009-04-09 09:39:48.000000000 +0200
+++ gdk-pixbuf/io-tiff.c	2009-04-09 09:40:03.000000000 +0200
@@ -134,7 +134,7 @@
         else {
                 g_set_error (error,
                              GDK_PIXBUF_ERROR,
-                             error_code, msg);
+                             error_code, "%s", msg);
         }
 }
 
--- gtk/gtkfilesystem.c.str	2009-04-09 09:42:30.000000000 +0200
+++ gtk/gtkfilesystem.c	2009-04-09 09:42:53.000000000 +0200
@@ -1020,7 +1020,7 @@
   fs_module->library = g_module_open (fs_module->path, 0);
   if (!fs_module->library)
     {
-      g_warning (g_module_error());
+      g_warning ("%s",g_module_error());
       return FALSE;
     }
   
@@ -1032,7 +1032,7 @@
       !g_module_symbol (fs_module->library, "fs_module_create", 
 			(gpointer *)&fs_module->create))
     {
-      g_warning (g_module_error());
+      g_warning ("%s",g_module_error());
       g_module_close (fs_module->library);
       
       return FALSE;
--- gtk/gtkimmodule.c.str	2009-04-09 09:43:12.000000000 +0200
+++ gtk/gtkimmodule.c	2009-04-09 09:43:30.000000000 +0200
@@ -92,7 +92,7 @@
   im_module->library = g_module_open (im_module->path, 0);
   if (!im_module->library)
     {
-      g_warning (g_module_error());
+      g_warning ("%s",g_module_error());
       return FALSE;
     }
   
@@ -106,7 +106,7 @@
       !g_module_symbol (im_module->library, "im_module_create", 
 			(gpointer *)&im_module->create))
     {
-      g_warning (g_module_error());
+      g_warning ("%s",g_module_error());
       g_module_close (im_module->library);
       
       return FALSE;
--- gtk/gtkthemes.c.str	2009-04-09 09:44:12.000000000 +0200
+++ gtk/gtkthemes.c	2009-04-09 09:44:29.000000000 +0200
@@ -80,7 +80,7 @@
   g_free(engine_path);
   if (!engine->library)
     {
-      g_warning (g_module_error());
+      g_warning ("%s",g_module_error());
       return FALSE;
     }
   
@@ -92,7 +92,7 @@
       !g_module_symbol (engine->library, "theme_create_rc_style", 
 			(gpointer *)&engine->create_rc_style))
     {
-      g_warning (g_module_error());
+      g_warning ("%s",g_module_error());
       g_module_close (engine->library);
       
       return FALSE;
--- tests/testmerge.c.str	2009-04-09 09:46:00.000000000 +0200
+++ tests/testmerge.c	2009-04-09 09:46:10.000000000 +0200
@@ -24,7 +24,7 @@
   gchar *dump;
 
   dump = gtk_ui_manager_get_ui (merge);
-  g_message (dump);
+  g_message ("%s",dump);
   g_free (dump);
 }
 
